### PR TITLE
Add content to Sunrise homepage

### PR DESCRIPTION
**Pull request by plaskevich**
Added static content to the homepage
Solve #118

### CODE REVIEW

File: src/components/home/PageHome.vue [hunk 1]
```diff
@@ -1,21 +1,54 @@
...
+    <div class="home-banner-option-three">
+      <div class="container-fluid">
+        <div class="row">
+          <BannerBottomLeft/>
+          <BannerBottomRight/>
+        </div>
+      </div>
+    </div>
 
     <div class="container">
```
Comment (lauraluiz) [inline src/components/home/PageHome.vue]: I would suggest to hide this `div.container` and anything inside, until we have more sections below.
 Without this div the page looks nicer :)

File: src/components/home/BannerMiddle.vue [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  description: "Die Modetrends für den Sommer 2019 spielen mit Prints, Army- und Safarilooks
+    sowie Pastel Farben. Accessoires wie runde Sonnenbrillen, geschnürte Stiefel und Shopper
+    oder Handtaschen sorgen für Abwechslung. SUNRISE stellt Ihnen die Fashion Trends für den
+    Sommer 2019 vor. Lassen Sie sich inspirieren! Kaum ist der Frühling vorbei,
+    umfängt uns der warme Sommer. Erfahren Sie, was die wichtigsten Fashion - Trends der
+    kommenden Sommersaison sind und welche traumhaften Stücke Sie besitzen sollen! London,
+    New York, Mailand, Paris – SUNRISE präsentiert die aktuelle Sommermode 2019 aus den Modemetropolen.
+    Die neuesten Kleider, elegante Tops, Hosen, Bademode, exklusive Schuhe und Ballerinas,
+    bezaubernden Schmuck, Sonnenbrillen und Accessoires für den perfekten Sommer."
+  linkLabel: "Shop collection"
```
Comment (lauraluiz) [inline src/components/home/BannerMiddle.vue]: Check on demo, this banner looks different from there:
https://demo.commercetools.com/

File: src/components/home/BannerBottomRight.vue [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+  subtitle2: "Accessories for the perfect summer"
+  subtitle3: "Hot pieces for the summer"
+  linkLabel: "Shop collection"
+de:
+  title1: "SOMMER MODE 2019"
+  title2: "Sonnenbrillen-Trends 2019"
+  title3: "Hauptsache überdimensional"
+  subtitle2: "Unsere Fashion Must-Haves"
+  subtitle3: "Trendy Sommer 2016 - Auffällige Sonnenbrillen"
+  linkLabel: "Shop collection"
```
[the remark this answers is not in the document]
Comment (lauraluiz) [reply src/components/home/BannerBottomRight.vue]: You can check the content in demo.commercetools.com if you want to see a version without years.

[the remark this answers is not in the document]
Comment (plaskevich) [reply src/components/home/BannerBottomRight.vue]: I still find years in demo :D but I will remove them anyway

**Pull request merged**